app/src/main/java/com/isc/hermes/model/searcher/Searcher.java

```diff
@@ -0,0 +1,78 @@
+package com.isc.hermes.model.searcher;
+
+
+import android.os.StrictMode;
+
+import com.isc.hermes.model.WayPoint;
+import com.mapbox.api.geocoding.v5.MapboxGeocoding;
+import com.mapbox.api.geocoding.v5.models.CarmenFeature;
+import com.mapbox.api.geocoding.v5.models.GeocodingResponse;
+import com.mapbox.geojson.Point;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import retrofit2.Response;
+
+
+/**
+ * Class to represent the searcher functionality of the app, setting the methods to make a call at the api to receive the real time suggestions of a query received
```

Comment:
it would be good that after the app you put an ENTER to refer to the title of the class.